pkg/controllers/raycluster_controller.go

```diff
@@ -177,31 +176,31 @@ func (r *RayClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			return ctrl.Result{RequeueAfter: requeueTime}, err
 		}
 
-		if isLocalInteractive {
-			logger.Info("Creating RayClient Route")
-			_, err := r.routeClient.Routes(cluster.Namespace).Apply(ctx, desiredRayClientRoute(&cluster), metav1.ApplyOptions{FieldManager: controllerName, Force: true})
-			if err != nil {
-				logger.Error(err, "Failed to update RayClient Route")
-				return ctrl.Result{RequeueAfter: requeueTime}, err
-			}
+		logger.Info("Creating RayClient Route")
+		_, err = r.routeClient.Routes(cluster.Namespace).Apply(ctx, desiredRayClientRoute(&cluster), metav1.ApplyOptions{FieldManager: controllerName, Force: true})
+		if err != nil {
+			logger.Error(err, "Failed to update RayClient Route")
+			return ctrl.Result{RequeueAfter: requeueTime}, err
 		}
 
 	} else if cluster.Status.State != "suspended" && !r.isRayDashboardOAuthEnabled() && !r.IsOpenShift {
-		ingressDomain := "" // TODO: ingressDomain should be retrieved by the CFO and used here to fix local_interactive. Jira: https://issues.redhat.com/browse/RHOAIENG-5330
+		logger.Info("We detected being on Vanilla Kubernetes!")
 		logger.Info("Creating Dashboard Ingress")
-		_, err := r.kubeClient.NetworkingV1().Ingresses(cluster.Namespace).Apply(ctx, desiredClusterIngress(&cluster, getIngressHost(ctx, r.kubeClient, &cluster, ingressDomain)), metav1.ApplyOptions{FieldManager: controllerName, Force: true})
+		dashboardName := dashboardNameFromCluster(&cluster)
+		dashboardIngressHost, _ := r.getIngressHost(ctx, r.kubeClient, &cluster, dashboardName)
```

Comment:
please process the error here, IMHO failing the reconciliation is a good approach

Reply:
Good call, will do thanks!